layout/TopNav.tsx

```diff
@@ -37,6 +33,20 @@ export const NAV_ITEMS: NavItem[][] = [
   ],
 ];
 
+if (getSiteVersion() === SITE_VERSION_HACKATHON_2023) {
```

Comment:
I generally prefer to write as much code outside of components as possible. If this causes any issues though, let me know.